src/components/CartItem/CartItem.tsx

```diff
@@ -0,0 +1,136 @@
+import { Link } from 'react-router-dom';
+import { CartItemType } from '../../types/CartItemType';
+import { calculatePrice } from '../../helpers/calculatePriceHelper';
+
+import './CartItem.scss';
+
+type Props = {
+  cartItem: CartItemType,
+  cart: CartItemType[],
+  setCart: (cart: CartItemType[]) => void,
+};
+
+export const CartItem: React.FC<Props> = ({ cartItem, cart, setCart }) => {
+  const transformType = (type: string) => {
+    switch (type) {
+      case 'phone':
+        return 'phones';
+
+      case 'tablet':
+        return 'tablets';
+
+      case 'accessory':
+        return 'accessories';
+
+      default:
+        return '';
+    }
+  };
+
+  const handleClickDeleteItem = (productId: string) => () => {
```

Comment:
```suggestion
  const handleDeleteItem = (productId: string) => () => {
```
and it's better to define these functions in the `CartPage` component and pass them as props. To achieve the single responsibility principle